configuration/concepts/history_and_examination.csv

```diff
@@ -0,0 +1,142 @@
+2a94ad77-778e-4075-58b7-68aa2c4926cc,,CIEL:124990,Tenesmus,Ténesme,Tenesmus,Ténesme,,,Misc,N/A,,,,,,,,,,,
+b792673c-24fe-5bd7-a165-0c30cbdf20cb,,CIEL:143264,Cough,Toux,Cough,Toux,,,Misc,N/A,,,,,,,,,,,
+f08dd30d-4211-50bf-6791-ab4d25e845a2,,CIEL:123074,Visual disturbance,Trouble visuel,Visual disturbance,Trouble visuel,,,Misc,N/A,,,,,,,,,,,
+6e6f3d4f-8bb9-7712-5d30-15ce540552b9,,CIEL:141831,Dizziness,Vertiges,Dizziness,Vertiges,,,Misc,N/A,,,,,,,,,,,
+ee87630f-8af9-57cd-532f-191579d287c6,,,Assault by animal suspected of rabies,Agression par animal suspect de rage,Assault by animal suspected of rabies,Agression par animal suspect de rage,,,Misc,N/A,,,,,,,,,,,
+2518a35f-a359-038e-55e0-63ddbc6078f3,,,Abnormal phenomenon,Phénomène anormal,Abnormal phenomenon,Phénomène anormal,,,Misc,N/A,,,,,,,,,,,
+24fb05d5-8447-a568-7d7b-f9f9cfdb08f6,,,Breathing difficulties,Difficultées respiratoires,Breathing difficulties,Difficultées respiratoires,,,Misc,N/A,,,,,,,,,,,
+33c31827-1f0d-4843-a84c-e98db2406b27,,CIEL:5219,Chief Complaint,Plainte principale,Chief Complaint,Plainte principale,,,Misc,Coded,,Fever;Swollen feet;Vision problems;Vaginal bleeding;Uterine contractions;Abdominal Lump;Anorexia;Anasarca;Anasarca;Anxiety disorder;Rectal bleeding;Chest Pain;Cleft Palate;Constipation;Cough;Infant crying excessively;Dental caries;Dysphagia;Edema;Epistaxis;Failure to thrive;Fatigue and malaise;Feeding problem(infant/elderly);Wheezing;Walking difficulty;Vomiting;Ulcer of lower limbs;Throat pain;Excessive sweating;Stomatitis and mucositis;Something coming out per vaginum;Sickle-cell anemia;Shortness of breath;Simple febrile seizures;Seizures/ convulsions;Something coming out of anus;Non vesicular rash;Polyuria;Palpitations;Backache;Knee pain;Trauma related chronic pain;Obstructed labor;Nocturia;Nausea;Nausea and vomiting;Mental status changes;Memory loss;Swelling;Breast lump;Localized superficial swelling/mass;Joint swelling;Irregular menstrual cycle;Injury;Infertility;Hoarseness;Hiccups;Hemoptysis;Hemiplegia and hemiparesis;Blood in vomit;Heartburn;Hearing loss;Headache;Gingival and periodontal diseases;Gas/bloating;Fracture;Abdominal pain;Tinnitus;Agitation;Altered consciousness;Amenorrhea;Aphasia;Arthralgia;Asthenia;Checkup;Voiding burns;Delirium;Watery diarrhea;Bloody diarrhea;Pain;Epigastric pain;Hypogastric pain;Dyspepsia;Dyspnea;Dysuria;Discharge from the eyes;Runny nose;Rashes;False needs;Chills;Halitosis;Hematuria;Hematemesis;Innapetence;Irritability;Lethargy;Leucorrhée;Lipothymie;Discomfort;Mass;Myalgia;Myasthenia;Nasal obstruction;Odinophagia;Otalgia;Otorrhea;Loss of consciousness;Weight loss;Wound;Pollakiuria;Polydipsia;Polyphagia;Hypertensive surge;Weight gain;Dermatological problems;Anal pruritus;Dermatological pruritus;Vaginal itching;Pyrosis;Rectorragie;Regurgitation;Delayed periods;Profuse sweats;Syncope;Tenesmus;Cough;Visual disturbance;Dizziness;Assault by animal suspected of rabies;Abnormal phenomenon;Breathing difficulties,,,,,,,,,

```

Comment:
Let's rename the French to 'Motif de consultation'

Reply:
Done